src/Analyzers/CSharp/CodeFixes/UseCollectionExpression/CSharpCollectionExpressionRewriter.cs

```diff
@@ -226,9 +239,10 @@ CollectionExpressionSyntax CreateCollectionExpressionWithExistingElements()
                     var braceIndentation = GetIndentationStringForToken(initializer.OpenBraceToken);
                     var preferredItemIndentation = initializer.Expressions.First().GetFirstToken().GetPreferredIndentation(document, indentationOptions, cancellationToken);
 
-                    initialCollection = initialCollection
-                        .WithElements(initialCollection.Elements.Replace(initialCollection.Elements.First(), initialCollection.Elements.First().WithLeadingTrivia(ElasticCarriageReturnLineFeed, Whitespace(preferredItemIndentation))))
-                        .WithCloseBracketToken(initialCollection.CloseBracketToken.WithLeadingTrivia(ElasticCarriageReturnLineFeed, Whitespace(braceIndentation)));
+                    initialCollection = initialCollection.Update(
+                        RemoveTrailingWhitespace(initialCollection.OpenBracketToken),
+                        FixLeadingAndTrailingWhitespace(initialCollection.Elements, preferredItemIndentation),
+                        initialCollection.CloseBracketToken.WithLeadingTrivia(endOfLine, Whitespace(braceIndentation)));
```

Comment:
note: this .Update and the .Update in the higher block.  I looked into unifying things, but felt it didn't help with clarity.